src/libsync/propagatedownloadzsync.cpp

```diff
@@ -190,6 +177,8 @@ void GETFileZsyncJob::seedFinished(void* zs) {
     }
     _propagator->reportFileTotal(*_item, zsyncSize);
 
+    _zs.release();
```

Comment:
That will make _zs drop its pointer without destroying it. Don't you still need it later? Where is it destroyed now?

Reply:
@ckamm again, please see latest push.